api/credentialmanager/client_test.go

```diff
@@ -25,7 +25,7 @@ func (s *CredentialManagerSuite) TestInvalidateModelCredential(c *gc.C) {
 	apiCaller := apitesting.APICallerFunc(func(objType string, version int, id, request string, arg, result interface{}) error {
 		c.Check(objType, gc.Equals, "CredentialManager")
 		c.Check(request, gc.Equals, "InvalidateModelCredential")
-		c.Assert(arg, gc.Equals, "")
+		c.Assert(arg, gc.Equals, params.InvalidateCredentialArg{Reason: ""})
```

Comment:
please have the test set a non empty reason to ensure it gets passed in correctly

Reply:
Sure, I'll modify one test to pass in non-empty string.